### PR TITLE
User can view entries - Cathy Garcia

### PR DESCRIPTION
**Pull request by cathyygrcia**
https://github.com/cathyygrcia/code-journal/assets/90016242/56a87c54-6f73-4fc8-bd11-ea7fc9506f96

### CODE REVIEW

File: css/layout.css [hunk 1]
```diff
@@ -29,6 +29,6 @@
   }
 
   .column-half {
-    width: 50%;
+    width: 49%;
   }
```
Comment (shawnkost) [inline css/layout.css]: This should stay at `50%`

File: css/styles.css [hunk 2]
```diff
@@ -23,6 +23,7 @@ h1 {
 
 img {
   width: 100%;
+  margin-bottom: 20px;
```
Comment (shawnkost) [inline css/styles.css:29]: Rather than giving this to the `img`, it should go to the `li` instead

File: css/styles.css [hunk 3]
```diff
@@ -71,3 +72,37 @@ button {
   padding: 0.3rem 0.7rem;
   text-transform: uppercase;
 }
+
+.hidden {
+  display: none;
+}
```
Comment (shawnkost) [inline css/styles.css]: This already exists in `layout.css`

File: js/main.js [hunk 4]
```diff
@@ -18,6 +19,73 @@ $entryForm.addEventListener('submit', function (event) {
...
+    $ul.appendChild($entry);
+  }
+});
+
+/* const $noEntries = document.querySelector('.no-entries')
+
+ function toggleNoEntries() {
+ $noEntries.classList.toggle('hide');
+
+} */
```
Comment (shawnkost) [inline js/main.js]: This shouldn't be commented out. This functionality should be working.

**Review by shawnkost — CHANGES_REQUESTED** [review]
Some changes here Cathy!

**Comment by cathyygrcia** [comment]
https://github.com/cathyygrcia/code-journal/assets/90016242/298be72e-0ca4-4fab-8c3c-af6a5951ae66

File: js/main.js [hunk 5]
```diff
@@ -60,12 +60,17 @@ document.addEventListener('DOMContentLoaded', function (event) {
...
- function toggleNoEntries() {
- $noEntries.classList.toggle('hide');
+function toggleNoEntries() {
+  if (data.entries.length > 0) {
+    $noEntries.classList.add('hidden');
+  } else {
+    $noEntries.classList.remove('hidden');
+  }
+}
+toggleNoEntries();
```
Comment (shawnkost) [inline js/main.js]: This should be getting called inside your `DOMContentLoaded` callback function & inside your `submit` callback function

**Review by shawnkost — CHANGES_REQUESTED** [review]
One more tweak here Cathy! This needs to be called elsewhere in a couple spots

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**